main.js

```diff
@@ -101,10 +100,16 @@ function getSelectedElementId() {
 
 const mouseUp = document.addEventListener('mouseup', function(e){
     const target = e.target
-    if (target.classList.contains("widget-box") && e.type === "mouseup") {
-        selectCell(target);        
-    } else if (target.nodeName !== "BUTTON") {
-        deselectCell();
+    if (target.id != "jsonLargeTextArea") {
```

Comment:
`!=` should be `!==`

Reply:
It also seems like this condition should be `(target.id !== "jsonLargeTextArea") && (e.type === "mouseup")` since the nested if statement is the only thing in this `if` block.

Reply:
👍